.github/workflows/release.yaml

```diff
@@ -0,0 +1,46 @@
+name: Release
+on:
```

Comment:
```
on:
  push:
    # Sequence of patterns matched against refs/tags
    tags:        
      - v*
```

Reply:
Okay I will use this, and remove these if sentences below.